### PR TITLE
BZ 1435382 Change service IP addresses for online and dedicated docs

### PR DESCRIPTION
**Pull request by ghost**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1435382

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@ahardin-rh Can you PTAL at the use of the ifdef/endif statement and the Labels for this PR? Thanks!

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -1089,16 +1090,48 @@ c8a648134623: Pushed
 latest: digest: sha256:be8bc4068b2f60cf274fc216e4caba6aa845fff5fa29139e6e7497bb57e48d67 size: 6273
 ----
 ====
+endif::openshift-enterprise[]
++
+ifdef::openshift-origin,openshift-dedicated[]
```
Comment (ahardin-rh) [inline dev_guide/managing_images.adoc]: According to Shawn, it looks like openshift-online needs to be included here too

File: dev_guide/managing_images.adoc [hunk 2]
```diff
@@ -1089,16 +1090,48 @@ c8a648134623: Pushed
...
+====
+----
+$ oc get is
+NAME       DOCKER REPO                        TAGS      UPDATED
+my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
+----
+====
+endif::openshift-enterprise[]
++
+ifdef::openshift-origin,openshift-dedicated[]
```
Comment (ahardin-rh) [inline dev_guide/managing_images.adoc]: looks like openshift-online is missing here too

**Comment by ahardin-rh** [comment]
@tmorriso-rh Since this file is shared with OCP, Online, and Dedicated, we should also make sure these changes are labeled for and cherry-picked to OCP 3.7, 3.9, and 3.10. (Dedicated and Online are both currently running on OCP 3.7. Tomorrow, Online will upgrade to 3.9.) So, even though the changes are mostly pertinent to Online and Dedicated, the files are still shared with OCP and so we have to make sure changes are applied in those branches as well. You are correct that no changes to the topic map are needed.

**Comment by ahardin-rh** [comment]
@tmorriso-rh LGTM! :+1:

**Pull request merged**